content/user-guide/certs-and-keys/certificates.md

```diff
@@ -0,0 +1,112 @@
++++
+title = 'Certificates and authentication'
+date = 2023-10-04T17:52:17-07:00
+draft = false
+weight = 31
++++
+
+- s2n_cert_auth_type
```

Comment:
I think I understand what you're doing here, where the APIs are used in the index to help the reader navigate to the correct section where they are discussed. I like it, although I think it makes our docs really difficult to read in the interim where we don't have a static site.

Reply:
The list of endpoints is to point out to us "what this page is about". I assume that we'll eventually remove it or hide it.